### PR TITLE
chore: Update CUDA base image to 12.3.2

### LINKED ISSUE #322: Support for CUDA 12.3.2
The other packages have put together support for 12.3.2 but this is the one package we need to make this upgrade.

When can we expect to see 12.3.2 support?

### PR DESCRIPTION
**Pull request by heckler1**
Fixes #322

### CODE REVIEW

**Comment by Talador12** [comment]
LGTM if this passes builds

File: versions.mk
Comment (elezar) [inline versions.mk:17]: Nit: Could we have the version bump as a separate commit?

**Review by elezar — CHANGES_REQUESTED** [review]

File: versions.mk
Comment (heckler1) [reply versions.mk:17]: Sounds good, I just split this into two commits. Thanks!

**Review by elezar — APPROVED** [review]
Thanks. LGTM

**Comment by heckler1** [comment]
Is there anything else you'd like me to do here before we merge?

**Comment by elezar** [comment]
@heckler1 we are in the process of migrating to GitHub and our actions didn't catch the fact that the CUDA 12.3.2 images are not yet available on dockerhub or NGC.

You're welcome to reopen / recreate this PR once the images are available.

@ArangoGutierrez we need to run at least one image build on PRs.

**Comment by heckler1** [comment]
Hey @elezar, sorry for the bad change, yeah happy to take another crack at this whenever images release.

**Pull request merged**
